packages/web/src/components/search/addons/SuggestionItem.js

```diff
@@ -0,0 +1,77 @@
+import React from 'react';
+import Highlight from 'react-highlight-words';
+import { css, cx } from 'react-emotion';
+
+import Flex from '../../../styles/Flex';
+
+const highlightStyle = {
+	fontWeight: 600,
+	padding: 0,
+	backgroundColor: 'transparent',
+	color: 'inherit',
+};
+
+const SuggestionItem = ({ currentValue, suggestion }) => {
+	const {
+		label,
+		value,
+		title,
+		description,
+		image,
+	} = suggestion;
+	if (label) {
+		// label has highest precedence
+		return typeof label === 'string'
+			? (
+				<div
+					className="trim"
+				>
+					<Highlight
+						searchWords={currentValue.split(' ')}
+						textToHighlight={label}
+						highlightStyle={highlightStyle}
+					/>
+				</div>
+			)
+			: label;
+	} else if (title || image || description) {
+		return (
+			<Flex alignItems="center" css={{ width: '100%' }}>
+				{
+					image
+					&& (
+						<div css={{ margin: 'auto', marginRight: 10 }}>
+							<img src={image} alt=" " height="50px" width="50px" css={{ objectFit: 'contain' }} />
```

Comment:
Would it better if we use `title` in `alt`?

Reply:
Actually this was intentional. Otherwise, while the images load you will see the alt attribute in the dropdown which looks weird while the suggestions' images are loading. With `' '` the space shows up as blank which is then filled by the image.